extensions.rst

```diff
@@ -0,0 +1,13 @@
+.. _extensions:
+
+Write a CPython extension
+=========================
+
+In this section, we could explain how to write a CPython extension with the C language, but the topic can take a complete book.
+
+For this reason, we prefer to give you some links where you can read a very good documentation.
+
+Read the following references:
+
+* https://docs.python.org/3.7/extending/
```

Comment:
For easier maintenance, use the `dev` link rather than the specific version: https://docs.python.org/dev/extending/